src/main/scala/com/typesafe/conductr/sbt/console/Column.scala

```diff
@@ -65,19 +69,33 @@ object Column {
     /**
      * Add '... ' if string is over the allowed width.
      */
-    private def ellipsize(s: String): String = {
+    protected def ellipsize(s: String): String = {
```

Comment:
Needed to be changed because we override this method in sub traits. Changed it to `private[console]`.